src/includes/getting-started-install/java.servlet.mdx

```diff
@@ -25,3 +25,13 @@ Sentry's Java SDK depends on [Gson](https://github.com/google/gson) as a transit
 If you are using multiple Sentry dependencies, you can add a [bill of materials](/platforms/java/configuration/bill-of-materials) to avoid specifying the version of each dependency.
 
 </Note>
+
+## Jakarta support
```

Comment:
I'd make it with `###` maybe to make it a child of the `Install` section, otherwise, it looks strange.

Reply:
The other option is, this should be at the end of this page: https://sentry-docs-git-feat-mention-sentry-servlet-jakarta.sentry.dev/platforms/java/guides/servlet/
and not in the install file.
Right now it's kinda in the middle of `install` and `configure` and people that don't care/know about Jakarta might get confused.

Reply:
I tried moving this further down. I couldn't do it. If I move into the next steps section my customizations didn't show up. If I move it into the Verify section I override the existing content. I didn't want to duplicate our `getting-started-verify/java.mdx` content.

I've changed the heading level. If we want another version I need help please.